src/components/ReanimatedSwipeable.tsx

```diff
@@ -317,10 +316,37 @@ const Swipeable = forwardRef<SwipeableMethods, SwipeableProps>(
       );
     };
 
+    const dispatchImmediateEvents = useCallback(
+      (fromValue: number, toValue: number) => {
+        if (toValue > 0 && props.onSwipeableWillOpen) {
+          props.onSwipeableWillOpen('left');
+        } else if (toValue < 0 && props.onSwipeableWillOpen) {
+          props.onSwipeableWillOpen('right');
+        } else if (props.onSwipeableWillClose) {
+          const closingDirection = fromValue > 0 ? 'left' : 'right';
+          props.onSwipeableWillClose(closingDirection);
+        }
+      },
+      [props.onSwipeableWillClose, props.onSwipeableWillOpen, swipeableMethods]
```

Comment:
I have to ask about this line.
My linter marks this as:

> unnecessary `prop.onSwipeableWillClose`, `props.onSwipeableWillOpen` deps

But isn't the current setup accommodating changes to those particular fields on the props?
These fields could change without triggering the broad `prop` dep as far as I understand.

Reply:
(in the latest commit, the `props` dep is not missing in the quoted code)

Reply:
I don't get any warnings locally, though I see that the code got changed in the meantime. It could've just lagged behind a bit and a reload would have solved it.

Reply:
After rebuilding i unfortunately still get the following error:
> React Hook useCallback has unnecessary dependencies: 'props.onSwipeableWillClose', 'props.onSwipeableWillOpen', and 'swipeableMethods'. Either exclude them or remove the dependency array.

I think I'll just have to `eslint-disable-next-line` it, because I am quite certain that these are in fact necessary.